qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/jupyterhub/files/jupyterhub/02-spawner.py

```diff
@@ -1,17 +1,11 @@
-# remove after next kubespawner release past 1/20/2022
-# https://github.com/jupyterhub/kubespawner/pull/558
+import json
 import os
 
-import kubernetes.client.models
-from tornado import gen
-
-kubernetes.client.models.V1EndpointPort = kubernetes.client.models.CoreV1EndpointPort
-
```

Comment:
Removing this line generated the following error:
```
 Loading extra config: 01-theme.py                                                                                                                                      │
│ Loading extra config: 02-spawner.py                                                                                                                                    │
│ [E 2022-09-05 18:46:15.302 JupyterHub app:2989]                                                                                                                        │
│     Traceback (most recent call last):                                                                                                                                 │
│       File "/opt/conda/lib/python3.7/site-packages/jupyterhub/app.py", line 2986, in launch_instance_async                                                             │
│         await self.initialize(argv)                                                                                                                                    │
│       File "/opt/conda/lib/python3.7/site-packages/jupyterhub/app.py", line 2477, in initialize                                                                        │
│         self.load_config_file(self.config_file)                                                                                                                        │
│       File "/opt/conda/lib/python3.7/site-packages/traitlets/config/application.py", line 110, in inner                                                                │
│         return method(app, *args, **kwargs)                                                                                                                            │
│       File "/opt/conda/lib/python3.7/site-packages/traitlets/config/application.py", line 889, in load_config_file                                                     │
│         raise_config_file_errors=self.raise_config_file_errors,                                                                                                        │
│       File "/opt/conda/lib/python3.7/site-packages/traitlets/config/application.py", line 844, in _load_config_files                                                   │
│         config = loader.load_config()                                                                                                                                  │
│       File "/opt/conda/lib/python3.7/site-packages/traitlets/config/loader.py", line 625, in load_config                                                               │
│         self._read_file_as_dict()                                                                                                                                      │
│       File "/opt/conda/lib/python3.7/site-packages/traitlets/config/loader.py", line 658, in _read_file_as_dict                                                        │
│         exec(compile(f.read(), conf_filename, "exec"), namespace, namespace)                                                                                           │
│       File "/usr/local/etc/jupyterhub/jupyterhub_config.py", line 446, in <module>                                                                                     │
│         exec(config_py)                                                                                                                                                │
│       File "<string>", line 5, in <module>                                                                                                                             │
│       File "/opt/conda/lib/python3.7/site-packages/kubespawner/__init__.py", line 13, in <module>                                                                      │
│         from kubespawner.spawner import KubeSpawner                                                                                                                    │
│       File "/opt/conda/lib/python3.7/site-packages/kubespawner/spawner.py", line 42, in <module>                                                                       │
│         from .objects import make_namespace                                                                                                                            │
│       File "/opt/conda/lib/python3.7/site-packages/kubespawner/objects.py", line 16, in <module>                                                                       │
│         from kubernetes.client.models import V1EndpointPort                                                                                                            │
│     ImportError: cannot import name 'V1EndpointPort' from 'kubernetes.client.models' (/opt/conda/lib/python3.7/site-packages/kubernetes/client/models/__init__.py) 
```

Which made the Hub deployment enter a crash loop cycle. Returning the block above fixed the problem.